tailnet/test/integration/network.go

```diff
@@ -390,33 +390,38 @@ func createFakeInternet(t *testing.T) fakeInternet {
 }
 
 type TriangleNetwork struct {
-	InterClientMTU int
+	Client1MTU int
 }
 
 type fakeTriangleNetwork struct {
-	NamePrefix             string
-	ServerNetNS            *os.File
-	Client1NetNS           *os.File
-	Client2NetNS           *os.File
-	ServerClient1VethPair  vethPair
-	ServerClient2VethPair  vethPair
-	Client1Client2VethPair vethPair
+	NamePrefix      string
+	ServerNetNS     *os.File
+	Client1NetNS    *os.File
+	Client2NetNS    *os.File
+	RouterNetNS     *os.File
+	ServerVethPair  vethPair
+	Client1VethPair vethPair
+	Client2VethPair vethPair
 }
 
-// SetupNetworking creates multiple namespaces with veth pairs between them
-// with the following topology:
+// SetupNetworking creates multiple namespaces with a central router in the following topology
 // .
-// .     ┌────────────────────────────────────────────┐
-// .     │                  Server                    │
-// .     └─────┬───────────────────────────────────┬──┘
-// .           │fdac:38fa:ffff:2::3                │fdac:38fa:ffff:3::3
-// .       veth│                               veth│
-// .           │fdac:38fa:ffff:2::1                │fdac:38fa:ffff:3::2
-// .   ┌───────┴──────┐                      ┌─────┴───────┐
-// .   │              │   fdac:38fa:ffff:1::2│             │
-// .   │  Client 1    ├──────────────────────┤  Client 2   │
-// .   │              │fdac:38fa:ffff:1::1   │             │
-// .   └──────────────┘                      └─────────────┘
+// .   ┌──────────────┐
+// .   │              │
+// .   │  Server      ├─────────────────────────────────────┐
+// .   │              │fdac:38fa:ffff:3::1                  │
+// .   └──────────────┘                                     │ fdac:38fa:ffff:3::2
+// .   ┌──────────────┐                               ┌─────┴───────┐
+// .   │              │            fdac:38fa:ffff:1::2│             │
+// .   │  Client 1    ├───────────────────────────────┤  Router     │
+// .   │              │fdac:38fa:ffff:1::1            │             │
+// .   └──────────────┘                               └─────┬───────┘
+// .   ┌──────────────┐                                     │ fdac:38fa:ffff:2::2
+// .   │              │                                     │
+// .   │  Client 2    ├─────────────────────────────────────┘
+// .   │              │fdac:38fa:ffff:2::1
+// .   └──────────────┘
+// The veth link between Client 1 and the router has a configurable MTU via Client1MTU.
 func (n TriangleNetwork) SetupNetworking(t *testing.T, l slog.Logger) TestNetworking {
```

Comment:
It's not really a triangle anymore unfortunately, more of a pitchfork 🤣

Reply:
You should redo the diagram to be triforce 🤣

Reply:
Not sure unicode has the correct (60°) diagonals